api/v1beta2/foundationdbcluster_types.go

```diff
@@ -950,6 +1028,11 @@ type AutomaticReplacementOptions struct {
 	// The default is 7200 seconds, or 2 hours.
 	FailureDetectionTimeSeconds *int `json:"failureDetectionTimeSeconds,omitempty"`
 
+	// TaintReplacementTimeSeconds controls how long a pod stays in NodeTaintReplacing condition
+	// before it is automatically replaced.
+	// The default is 1800 seconds, i.e., 30min
+	TaintReplacementTimeSeconds *int `json:"taintReplacementTimeSeconds,omitempty"`
```

Comment:
What is the benefit of adding a new setting and not using `FailureDetectionTimeSeconds`?